com.unity.multiplayer.transport.utp/Runtime/UTPTransport.cs

```diff
@@ -278,10 +278,10 @@ public override unsafe void Send(ulong clientId, ArraySegment<byte> data, Networ
         SendToClient(writer.AsNativeArray(), peerId, pipelineIndex);
     }
 
-    public override NetcodeEvent PollEvent(out ulong clientId, out NetworkChannel networkChannel, out ArraySegment<byte> payload, out float receiveTime)
+    public override NetcodeEvent PollEvent(out ulong clientId, out NetworkDelivery networkDelivery, out ArraySegment<byte> payload, out float receiveTime)
     {
         clientId = 0;
-        networkChannel = NetworkChannel.ChannelUnused;
+        networkDelivery = NetworkDelivery.ReliableSequenced; //??
```

Comment:
Do we actually need to pop this now? because we did this before so the NetworkManager would know the channel but thats not required anymore?